platform/commonUI/general/res/sass/openmct.scss

```diff
@@ -0,0 +1,39 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+.app-logo {
+    background: url('/platform/commonUI/general/res/images/logo-app.svg') no-repeat center center;
+}
+
+.l-splash-holder {
+    background: #333;
+    .s-splash {
+        border-radius: 10px;
+        box-shadow: 0 5px 50px 25px rgba(255, 255, 255, 0.1);
+    }
+}
+
+.s-splash {
+    background-image: url('/platform/commonUI/general/res/images/bg-splash.jpg');
+    &:after {
+        background-image: url('/platform/commonUI/general/res/images/logo-app-shdw.svg');
+    }
```

Comment:
I thought Pete told me that root-based URLs get dynamically interpreted, i.e. /platform/commonUI/... resolves to both that as well as /node_modules/openmctweb/platform/commonUI/... I may well have misunderstood though. Not a problem in open, but I didn't want to use relative paths in VISTA and WARP to get to open resources.